src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {
```

Comment:
```suggestion
public enum FigureName {
```
enum names should be in singular